docs/how-to/build-policies/abac/temporary_role.mdx

```diff
@@ -0,0 +1,145 @@
+    <img src="/img/abac/time_based_groups.png" style={{ maxHeight: "45vh" }} />
+  1. Add `time_based_groups` to specific users.
+    <img src="/img/abac/time_based_groups_user.png" style={{ maxHeight: "45vh" }} />
+  1. Make sure the user is [assigned the Top Level Role in the Directory](https://app.permit.io/user-management) as well:
+    <img src="/img/abac/time_based_user_assignment.png" style={{ maxHeight: "45vh" }} />
+
+
+  ```javascript
+  "time_based_groups": [
+    {
+      "role": "admin",
+      "tenant": "default",
+      "expires": 1733256879832
+    },
+    {
+      "role": "admin",
+      "tenant": "coke",
+      "expires": 1733861761081
+    }
+  ]
+  ```
+:::info
+The `expires` value above is the Unix timestamp in milliseconds. This requires that the `current_time` value is passed in the `permit.check()` as a user attribute.
+:::
+
+
+- ### Create condition set
+
+  The condition set checks three key elements:
+  - If the user has the admin role
+  - If the expiration time hasn't passed
+  - If the tenant key matches
+
+  Here's an example API call to create the condition set:
+  ```bash
+  curl --location 'https://api.permit.io/v2/schema/default/production/condition_sets' \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer permit_key_' \
+  --data '{
+    "key": "temp-admin",
+    "name": "Temporary Admin",
+    "type": "userset",
+    "conditions": {
+      "allOf": [
+        {
+          "allOf": [
+            {
+              "user.roles": {
+                "array_contains": "admin"
+              }
+            },
+            {
+              "user.time_based_groups": {
+                "any_match": {
+                  "match": {
+                    "expires": {
+                      "greater-than-equals": {
+                        "ref": "user.current_time"
+                      }
+                    },
+                    "tenant":{
+                      "equals": {
+                        "ref": "tenant.key"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          ]
+        }
+      ]
+    }
+  }'
+  ```
+
+- ### Create condition set
+    1. Assign the appropriate resource actions to the new "temp-admin" role through the [Policy Editor](https://app.permit.io/policy-editor) after creating the condition set.
+        <img src="/img/abac/permit_ui_temp_admin.png" style={{ maxHeight: "45vh" }} />
+
+- ### Test the user using `permit.check()`
+    ```javascript
+    const permitted = await permit.check(
+    // the user object
+    {
+        // the user key
+        key: "george@test.com",
+        // just-in-time attributes on the user
+        attributes: {
+            current_time: 1734635473238 //needs to be generate at check time
```

Comment:
Maybe set the current time as 'datetime.now()' or some other method that makes sense that it's a programmatic value